src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -191,12 +191,13 @@ def _get_regex_callbacks(self) -> list[RegexCallback]:
 
             completed_regexes = [re.compile(".*Finished Rendering.*")]
             progress_regexes = [re.compile(".*Progress ([0-9]+)%.*")]
-            error_regexes = [re.compile(".*Error: .*|.*\\[Error\\].*", re.IGNORECASE)]
+            error_regexes = [
+                re.compile(".*Error: .*|.*\\[Error\\].*|.*CRITICAL: .*", re.IGNORECASE)
```

Comment:
Also I wonder if we should add regex for [CRITICAL] as well similar to [ERROR] pattern checking?

 Something like `.*Error: .*|.*\\[Error\\].*|.*CRITICAL: .*|.*\\[Critical\\].*`? Because if the log line does not have `CRITICAL: ` (with space at the end) then this would not fail the job.

Reply:
Hmm, never mind. I searched for logs with [CRITICAL] and I couldn't find any.